### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by RachelGreenwood**
Completed:
-Adds unit tests
-Makes quiz buttons and feedback work
-Some CSS styling
-Puts text-to-speech API call in Section component instead of Lesson to make it easier for the user
-Adds new DB schema
-Worked on README

Still Need To:
-Add integration tests
-Add content instead of placeholder text
-Finish CSS styling
-Fix error in navbar (lesson one shows up twice; the 2nd iteration is correct, the 1st holds no data???)

### CODE REVIEW

File: client/src/App.test.js [hunk 1]
```diff
@@ -1,8 +1,6 @@
-import { render, screen } from '@testing-library/react';
+import { render } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
+test('renders App component', () => {
   render(<App />);
```
Comment (Technically-Teta) [inline client/src/App.test.js:7]: Even though you have a test file would it be beneficial to add a comment to say the test passed here? For Stem review purposes?

**Comment by Technically-Teta** [comment]
Great Job Rachel! All tests look good and you are on track!

File: client/src/App.test.js (hunk 1 above)
Comment (RachelGreenwood) [reply client/src/App.test.js:7]: Great idea, thank you Samelia!

**Review by Technically-Teta — COMMENTED** [review]
You should be good to go to merge, are you all set to do so?

**Review by frankstepanski — APPROVED** [review]
Looks good. Don't see any glaring issues. I would say when your app grows start putting your components in separate folders and start adding styling per component.

**Comment by RachelGreenwood** [comment]
> Looks good. Don't see any glaring issues. I would say when your app grows start putting your components in separate folders and start adding styling per component.

Thank you, Frank! I just put my components in thematically-grouped folders. I'll start adding styling files for each component soon when I add more styling rules!

**Pull request merged**